shapely/geometry/geo.py

```diff
@@ -118,7 +119,7 @@ def shape(context):
         raise GeometryTypeError(f"Unknown geometry type: {geom_type!r}")
 
 
-def mapping(ob):
+def mapping(ob, force_ccw=False):
```

Comment:
@rkaravia I'd like to see `ensure_ccw` or `orient_ccw` as the name of this keyword argument. Force, to me, means to push past warnings and ignore security or safety checks. We're not doing that here. We're only calling another method, which is named `orient_polygons()`.